source/module.c

```diff
@@ -22,12 +22,22 @@
 
 #include <aws/mqtt/mqtt.h>
 
+#if PY_MAJOR_VERSION == 3
+#    define INIT_FN PyInit__aws_crt_python
+#    define UNICODE_GET_BYTES_FN PyUnicode_DATA
+#    define UNICODE_GET_BYTE_LEN_FN PyUnicode_GET_LENGTH
```

Comment:
This returns the number of "Code Points" in the string, not the number of bytes in the raw data